opmon/performance-review-kenya.toml

```diff
@@ -0,0 +1,469 @@
+description = "The upload throughput for http/2 requests larger than 10MB. Measured in megabits per second, Mbps.\nBreakdown by country."
+metrics = [
+    "networking_http_2_upload_throughput_KE",
+    "networking_http_2_upload_throughput_US"
+]
+
+[project.metric_groups.networking_http_3_upload_throughput]
+friendly_name = "HTTP 3 Upload Throughput by country"
+description = "The upload throughput for http/3 requests larger than 10MB. Measured in megabits per second, Mbps.\nBreakdown by country."
+metrics = [
+    "networking_http_3_upload_throughput_KE",
+    "networking_http_3_upload_throughput_US"
+]
+
+[project.metric_groups.performance_time_response_start]
+friendly_name = "Time Response Start by country"
+description = "Time from navigationStart to responseStart as per the W3C Performance Timing API.\nBreakdown by country."
+metrics = [
+    "performance_time_response_start_KE",
+    "performance_time_response_start_US"
+]
+
+[project.metric_groups.performance_pageload_load_time]
+friendly_name = "Pageload Load Time by country"
+description = "Time in milliseconds from navigationStart to loadEventStart for the foreground http or https root content document.\nBreakdown by country."
+metrics = [
+    "performance_pageload_load_time_KE",
+    "performance_pageload_load_time_US"
+]
+
+[project.metric_groups.perf_startup_cold_main_app_to_first_frame]
+friendly_name = "Startup Cold Main App to First Frame by country"
+description = "The duration from `*Application`'s initializer to the first Android frame\nbeing drawn in a [COLD MAIN start\nup](https://wiki.mozilla.org/index.php?title=Performance/Fenix/Glossary).\nBreakdown by country."
+metrics = [
+    "perf_startup_cold_main_app_to_first_frame_KE",
+    "perf_startup_cold_main_app_to_first_frame_US"
+]
+
+[project.metric_groups.perf_startup_cold_view_app_to_first_frame]
+friendly_name = "Startup Cold View App to First Frame by country"
+description = "The duration from `*Application`'s initializer to the first Android frame\nbeing drawn in a [COLD VIEW start\nup](https://wiki.mozilla.org/index.php?title=Performance/Fenix/Glossary).\nBreakdown by country."
+metrics = [
+    "perf_startup_cold_view_app_to_first_frame_KE",
+    "perf_startup_cold_view_app_to_first_frame_US"
+]
+
+
+[data_sources]
+
+[data_sources.baseline_US_and_KE]
+
+from_expression = """(
+    SELECT
+        p.*,
+        DATE(p.submission_timestamp) AS submission_date
+    FROM `moz-fx-data-shared-prod.{dataset}.baseline` p
+    WHERE metadata.geo.country in ("US", "KE")
+)"""
+
+client_id_column = "client_info.client_id"
+experiments_column_type = "glean"
+default_dataset = "org_mozilla_firefox"
+friendly_name = "Baseline"
+description = "Baseline Ping"
+build_id_column = "REPLACE(CAST(DATE(mozfun.norm.fenix_build_to_datetime(client_info.app_build)) AS STRING), '-', '')"
+
+
+[metrics]
+
+# Storage Stats App Bytes
+[metrics.storage_stats_app_bytes_KE]
+data_source = "metrics"
```

Comment:
it might also help to sample the `metrics` data source since this will be joined onto the population